lib/model.py

```diff
@@ -90,3 +91,55 @@ class TextModel:
 
 class ImageModel:
     pass
+
+
+class Evaluator:
+    def __init__(self, model_type):
+        self.model_type = model_type
+    
+    def evaluate(self, y_true, y_predict, more=False):
+        def data_is_valid():
+            return len(y_true) == len(y_predict)
+        
+        def evaluate_classification():
+            scores = {
+                'Accuracy': accuracy_score(y_true, y_predict),
```

Comment:
Updated. Not sure about recallAt1 part.

Reply:
> Updated. Not sure about recallAt1 part.
The metrics supported in the implementation could be a subset of the metrics in OpenAPI doc, if you doesn't have a plan to support recallAt1, please just remove it in your implementation instead of assigning a fixed value to this metric.